docs/line.asciidoc

```diff
@@ -27,9 +27,27 @@ NOTE: In Elasticsearch releases 1.4.3 and later, this functionality requires you
 {ref}/modules-scripting.html[dynamic Groovy scripting].
 
 The availability of these options varies depending on the aggregation you choose.
-Select *view options* to change the following aspects of the table:
 
+Select the *Options* tab to change the following aspects of the chart:
+
+*Y-Axis Scale*:: You can select *linear*, *log*, or *square root* scales for the chart's Y axis.
```

Comment:
Added this section to the discussion of different axis scales: "You can use a log 
scale to display data that varies exponentially, such as a compounding interest chart, or a square root scale to regularize the display of data sets with variabilities that are themselves highly variable. This kind of data, where the variability is itself variable over the domain being examined, is known as _heteroscedastic_ data. For example, if a data set of height versus weight has a relatively narrow range of variability at the short end of height, but a wider range at the taller end, the data set is heteroscedastic."